### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by pratikw007** [community/community#31006] [Code Security]
Deceptive site ahead
Attackers on pratikw007.github.io may trick you into doing something dangerous like installing software or revealing your personal information (for example, passwords, phone numbers, or credit cards). [Learn more](chrome-error://chromewebdata/#)

Why We get an Error??

**Comment by radar-blip** [comment]
Check accepted answer here: #30261  
special thanks to airtower-luna 😃
